showcase/neic_pari.md

```diff
@@ -0,0 +1,351 @@
+---
+title: The NeIC PaRI partners’ guide to SARS-CoV-2 genome data sharing
+description: 
+---
+Produced by the Nordic Pandemic Research Infrastructure project (NeIC PaRI)
+[https://neic.no/pari/](https://neic.no/pari/)
+
+This document aims to 1) outline what constitutes a good and ultimately (re)usable viral genome data record in a data repository with a Nordic perspective; and 2) provide guidance to projects, labs and other organisations producing or commissioning viral sequencing data.
+
+## Introduction
```

Comment:
1. SPLIT SUGGESTION: Could generalise this to be more of an introduction about data description in the pathogen characterisation section